release-docs/files/run.sh

```diff
@@ -18,10 +18,20 @@ echo ""
 mkdir -p objs
 VERSION=${VERSION} doxygen
 
+if [ -e "CMakeLists.txt" ]; then
+    (
+        mkdir build
+        cd build
+        cmake ..
+        make script_window || true
+    )
+    GENERATED_API_DIR=../../../build/generated/script/api
```

Comment:
why not make this an absolute path? Something like `$(pwd)/generated/script/api`?